core/Updates/2.15.1-b1.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Common;
+use Piwik\Updater;
+use Piwik\Updates;
+
+
+class Updates_2_15_1_b1 extends Updates
+{
+    public function getMigrationQueries(Updater $updater)
+    {
+        $updateSql = array(
+            'ALTER TABLE `' . Common::prefixTable('user_language')
```

Comment:
I was also not sure last time I had a similar case. I think we should but the update statements were the install statements are. So if the table is defined in `Core/Schema/MySQL` we should put the updates into `Core` as well. If the MySQL table is defined in a plugin (via `install`) the updates should be defined in the plugin itself